docs/deployer_config.md

```diff
@@ -6,6 +6,20 @@
 
 AWS profile to use for the deployment. Also used for all AWS CLI commands run locally. See the [AWS docs](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) for more information.
 
+## AWSRegion
+
+*string*
+
+AWS region to use for the deployment.  See the [AWS docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) for more information.
+
+## AWSAMI
+
+*string*
+
+AWS AMI to use for the deployment. This is the image used for all EC2 instances created by the loadtest tool. See the [AWS AMI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) docs for more information. We suggest Ubuntu 20.04 or 22.04. Note, the AMI could change between AWS Regions.
```

Comment:
An additional comment, sorry for not noticing this earlier. This:

> Note, the AMI could change between AWS Regions.

reads a bit weird to me. It may be my non-native English, but to me this seems to mean that the same AMI ID may point to different AMIs in different regions. While I assume you wanted to mean that you may need to change the AMI based on the region the configuration uses, since it may not be available in the region.

Did I misunderstood the comment?

Reply:
I meant to say that there is a chance your AMI exists across regions in AWS (In reality I have zero idea, since there are a ton of AMIs / Regions). I don't think it would point to an alternate OS between regions though